command.c

```diff
@@ -1082,6 +1082,10 @@ bool command_event_save_auto_state(
    bool ret                    = false;
    char savestate_name_auto[PATH_MAX_LENGTH];
 
+#ifdef HAVE_ENTRYSTATES
+   if (!path_is_empty(RARCH_PATH_STATE))
```

Comment:
I like your use of the term 'entry state' (as in entry point) - and I believe it should be used everywhere in the code as well (since just using/suffixing 'state' is a little too generic, and could refer to other things)

So I'd suggest changing `RARCH_PATH_STATE` to `RARCH_PATH_ENTRY_STATE`

Reply:
I switched to slot indices instead of paths. It fits in nicely with the way you handle save states already. Entry states will be put alongside save states as "content_name.state42.entry". So everything RARCH_PATH_STATE is gone and the rest is names according to your suggestions